autobot-backend/knowledge/facts_metadata_sync_test.py

```
@@ -19,13 +19,27 @@
 
 import json
-from unittest.mock import AsyncMock, MagicMock
+from unittest.mock import AsyncMock, MagicMock, patch
 
 import pytest
 
 from knowledge.facts import FactsMixin
 
 
+@pytest.fixture(autouse=True)
+def _durable_row_exists():
```

Ruff (0.16.3)

[warning] Missing return type annotation for private function `_durable_row_exists`

(ANN202)